osx/pre_installation.sh

```diff
@@ -92,6 +87,12 @@ function check_kalite_exc_collector {
 ENV=$(env)
 syslog -s -l error "Packages pre-installation initialize with env:'\n'$ENV" 
 
+# Collect the directories and files to remove
+append REMOVE_FILES_ARRAY $HOME_LAUNCH_AGENTS/$KALITE_PLIST
+append REMOVE_FILES_ARRAY $ROOT_LAUNCH_AGENTS/$KALITE_PLIST
+append REMOVE_FILES_ARRAY $KALITE_RESOURCES
+append REMOVE_FILES_ARRAY $KALITE_MONITOR
+
```

Comment:
The same as `KA-Lite_Uninstall.tool` - don't assume that those files or directories exists! Always check they actually exist on the target computer before adding them to the array.